src/rgw/rgw_op.cc

```diff
@@ -2538,10 +2538,9 @@ void RGWGetUsage::execute()
 
   RGWUsageIter usage_iter;
   
-  while (is_truncated) {
-    op_ret = store->getRados()->read_usage(s->user->get_id(), s->bucket_name, start_epoch, end_epoch, max_entries,
```

Comment:
@ofriedma  I've been trying to discern whether this particular change actually modifies functionality. As far as I can tell, it's just using a more modern API (not available in Octopus) to achieve the same thing. Can you confirm whether this change can be omitted in Octopus?